src/compiler/transformers/ts.ts

```diff
@@ -916,7 +916,7 @@ namespace ts {
                 );
 
                 const parameters = transformConstructorParameters(constructor);
-                const body = transformConstructorBody(node.members, constructor, parametersWithPropertyAssignments);
+                const body = transformConstructorBody(existingMembers, constructor, parametersWithPropertyAssignments);

```

Comment:
TL;DR: `node.members` are the original, untransformed members; `existingMembers` have been transformed through `visitTypeScript`.